### ISSUE TITLE
[Question] What is the best way to monitor metrics from kube-prometheus?

### ISSUE THREAD

**Issue by jason-riddle** [coreos/prometheus-operator#263]
The Grafana dashboards provided do a great job as far as visualizing your metrics. Since alerting is now available in Grafana (as of v4.0) are there any recommended thresholds to alert on? Even a small document with some recommendations on what to alarm on would be great.

**Comment by brancz** [comment]
We do not recommend alerting with Grafana. We ship with alerts out of the box using Prometheus alerting rules and a highly available Alertmanager cluster, which Grafana alerting is far away from. I recommend reading our documentation on the [Prometheus/Alertmanager high availability scheme](https://github.com/coreos/prometheus-operator/blob/master/Documentation/high-availability.md). 

tl;dr with Grafana when you have a highly available cluster you will always get number of grafana instances as alert notifications. Alerting with Prometheus and Alertmanager on the other hand has a well thought through HA model.

I'll close this here, but if you have any further questions or something you would like us to clarify on let us know and we'll be happy to answer :slightly_smiling_face: .

**Comment by jason-riddle** [comment]
Derp, forgot about alertmanager. Thanks!
